packages/appsync-emulator-serverless/__test__/tester.test.js

```diff
@@ -122,4 +151,265 @@ describe('appsync-emulator-serverless/tester', () => {
     },
     40000,
   );
+
+  it('subscription with no arguments', async () => {
```

Comment:
broke these up because a client can only subscribe to on topicId at a time

Reply:
Hrrm is that a bug ? I am sure we should be able to handle multiple subscriptions in a working case.

Reply:
My comment isn't clear.

If a client wants to subscribe to a topic, and is already subscribing to that topic, the previous subscription is removed and the new subscription takes its place.  Therefore, instead of creating N clients in a single unit test which all subscribe to the same topicId, I broke up the unit tests to handle a separate subscription use case so that 1-2 clients were necessary.